### DISCUSSION TITLE
CTS602 with LAN Gateway and Home Assistant Integration? (Compact P)

### DISCUSSION THREAD

**Discussion by greg2011** [veista/nilan#145] [Q&A] [answered]
Hi all!
I hope you can help, I'm a bit lost here...

My Compact P / CTS602 is currently both connected to a remote touch panel (in the living area) and a LAN Gateway. - See pictures.

I now would like to keep my (remote) touch panel and use home assistant - I understand this could fully replace my Nilan App, but there's no possibility / need to use gateway and home assistant in parallel?

Can I do this with a waveshare converter and some changes to my cabling (using the existing wires that go into my gateway)?
Or is there anything else that I need to do?

Thanks a lot,
Greg

![20240913_130719](https://github.com/user-attachments/assets/cd3c4ac0-ae1c-48e4-86da-f3d24891839a)
![20240913_131539](https://github.com/user-attachments/assets/26f6a6a0-da6f-4c5e-baeb-d16d7a068f8b)
![20240913_130945](https://github.com/user-attachments/assets/5105ee48-4d6b-4f51-818e-69d4e2fb607d)

**Comment by veista** [comment]
You can't use the official gateway at the same time unfortunately. You need to remove the gateway and replace it with a supported bridge device or serial interface.

**Reply by veista** [reply]
Ps. There should be no need to.

**Comment by veista** [comment]
https://github.com/veista/nilan/wiki/Wiring-for-CTS602

Here you can find the wiring.

**Comment by veista** [comment, marked as answer]
The display works normally with cts602.

**Comment by greg2011** [comment]
WOW that was a quick reply and you answered all my questions:
- I really don't need the gateway anymore
- can re-wire with the existing cabling
- and the remote display should still work

Thank you!

**Reply by veista** [reply]
No problem. You are most welcome.
